packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -41,49 +41,27 @@ describe('plugin-conversation', () => {
         const {conversation} = webex.internal;
         const recipientId = 'example-recipient-id';
         const expected = {items: [{id: recipientId, objectType: 'person'}]};
+        conversation.config.includeEncryptionTransforms = true;
         conversation.sendReaction = sinon.stub().returns(Promise.resolve());
         conversation.createReactionHmac = sinon.stub().returns(Promise.resolve('hmac'));
 
         return conversation.addReaction({}, 'example-display-name', {}, recipientId).then(() => {
+          assert.called(conversation.createReactionHmac);
           assert.deepEqual(conversation.sendReaction.args[0][1].recipients, expected);
         });
       });
-    });
-
-    describe('deleteReaction()', () => {
```

Comment:
there was duplicated code here and the exact same test blocks were here twice. i deleted the duplicated blocks